### ISSUE TITLE
Profitability Level

### ISSUE THREAD

**Issue by bwstandard** [RainbowMiner/RainbowMiner#2560]
Just wanted some clarity on the below

ProfitabilityLevel = profitability level in BTC in case CheckProfitability is set to 1 (e.g. -0.00002) [default=0]

I always having the miners checking for profit, however it would be nicer if I could have them only turn on at a certain amount. Right now I just turn machines off at the power switch and then turn them back on via phone and software. If I put in a btc value then they will stop mining if below that level? 

I don't want to whole farm running if profit is too low. 

Thanks

**Comment by RainbowMiner** [comment]
> If I put in a btc value then they will stop mining if below that level?

Yes. If below, RainbowMiner will loop until the profitability rises above that level again.
